File: Exercícios/PythonMundo_1_2_3/Desafio_08.py
# Exercício Python 8: Escreva um programa que leia um valor em metros e o exiba convertido em centímetros e milímetros.

print('{} DESAFIO 08 {}'.format('='*6, '='*6))

metros = float(input('Digite um valor em metros: '))
quilometros = metros * 0.001
hectometro = metros * 0.01
decametros = metros * 0.1
decimetros = metros * 10
centimetros = metros * 100
milimetros = metros * 1000

print(f'\n[!] METROS = {metros:.2f}m\n[!] QUILÔMETROS = {quilometros:.3f}km')
print(f'[!] HECTÔMETROS = {hectometro:.2f}hm\n[!] DECÂMETROS = {decametros:.2f}dam\n[!] DECÍMETROS = {decimetros:.2f}dm')
print(f'[!] CENTÍMETROS = {centimetros:.2f}cm\n[!] MILÍMETROS = {milimetros:.2f}mm')

File: Exercícios/Scripts_C#/ScriptsUsandoC#/ScriptsUsandoC#/Desafio_08.cs
public class Desafio_08
{
	public static void Distancias()
	{
		Console.Write("Digite um valor em Metros: ");
		double metros = double.Parse(Console.ReadLine().Trim().Replace(".", ","));

		double quilometros = metros * 0.001;
		double hectometro = metros * 0.01;
		double decametros = metros * 0.1;
		double decimetros = metros * 10;
		double centimetros = metros * 100;
		double milimetros = metros * 1000;

		Console.WriteLine($"O valor em Quilômetros é {quilometros:F3}");
		Console.WriteLine($"O valor em Hectômetros é {hectometro:F2}");
		Console.WriteLine($"O valor em Decâmetros é {decametros:F2}");
		Console.WriteLine($"O valor em Decímetros é {decimetros:F2}");
		Console.WriteLine($"O valor em Centímetros é {centimetros:F2}");
		Console.WriteLine($"O valor em Milímetros é {milimetros:F2}");

    }
}
